client/src/pages/Home.jsx

```diff
@@ -26,6 +28,10 @@ function Home({ user }) {
       {sessions.map((session, i) => (
         <SessionSummary session={session} key={i} />
       ))}
+      <SessionForm
+        button={<Button type="default">+</Button>}
+        id={user.memberDbId}
```

Comment:
use `user._id` instead bc that's what we're using to fetch sessions as well. `_id` is the mongoDB unique identifier, while `memberDBId` is the ID given by the memberDB tool.